stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1801,7 +1801,7 @@ function(add_swift_target_library name)
   if (SWIFTLIB_IS_STDLIB)
     list(APPEND SWIFTLIB_SWIFT_COMPILE_FLAGS "-warn-implicit-overrides")
     list(APPEND SWIFTLIB_SWIFT_COMPILE_FLAGS "-Xfrontend;-enable-ossa-modules")
-    list(APPEND SWIFTLIB_SWIFT_COMPILE_FLAGS "-Xfrontend;-enable-lexical-lifetimes=false")
+    list(APPEND SWIFTLIB_SWIFT_COMPILE_FLAGS "-Xfrontend;-enable-lexical-lifetimes=true") # FIXME: undo this
```

Comment:
TODO: We can't include @_moveOnly code in stdlib, so this is just to allow experimentation.

If we decide to go with it, we could add the annotations later on and just land this "ready for move-ification" I suppose.